src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type Props = {
+  comment: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentsInfo__title">
+      <strong className="CommentInfo__name">{comment?.name}</strong>
```

Comment:
```suggestion
      <strong className="CommentInfo__name">{comment.name}</strong>
```
Comment is required props and can't be undefined based on your Props type